### PR TITLE
feat: add encoding to update verb and update verb builder

### PR DESCRIPTION
**Pull request by sitaram-kalluri**
**- What I did**
- Add encoding to the update verb regex and update verb builder
- Add encoding to the Metadata
- Add unit tests to validate the above changes

**- Description for the changelog**
- Add encoding to support encoding of new line characters

### CODE REVIEW

File: at_commons/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+## 3.0.22
+- Add 'encoding' to update verb builder to support encoding of new line character
```
Comment (gkc) [inline at_commons/CHANGELOG.md]: Also added to Metadata

File: at_commons/lib/src/keystore/at_key.dart [hunk 2]
```diff
@@ -332,6 +334,7 @@ class Metadata {
       metaData.sharedKeyStatus = json[SHARED_KEY_STATUS];
       metaData.sharedKeyEnc = json[SHARED_KEY_ENCRYPTED];
       metaData.pubKeyCS = json[SHARED_WITH_PUBLIC_KEY_CHECK_SUM];
+      metaData.encoding = json['encoding'];
```
Comment (gkc) [inline at_commons/lib/src/keystore/at_key.dart]: change 'encoding' to a constant and declare it in the same place that SHARED_KEY_ENCRYPTED etc are declared

**Review by gkc — CHANGES_REQUESTED** [review]

File: at_commons/lib/src/keystore/at_key.dart (hunk 2 above)
Comment (sitaram-kalluri) [reply at_commons/lib/src/keystore/at_key.dart]: Added constant in for 'encoding'

File: at_commons/CHANGELOG.md (hunk 1 above)
Comment (sitaram-kalluri) [reply at_commons/CHANGELOG.md]: Updated the changelog.

**Review by gkc — APPROVED** [review]

**Pull request merged**